edge-modules/api-proxy-module/docker/linux/arm64v8/Dockerfile

```diff
@@ -5,17 +5,18 @@
 
 ARG DIR=.
 
-FROM arm64v8/alpine:3.12.0
+FROM arm64v8/nginx:1.19.2
```

Comment:
why change from hash to tag? how does it works as amd64 agent is used to build arm64?

Reply:
It uses musl builder, a cross build image